### ISSUE TITLE
[QUESTION] what is edk2-nvidia and changing UEFI variables for Jetson?

### ISSUE THREAD

**Issue by PHCLenzi** [NVIDIA/edk2-nvidia#74]
Hello, two questions here

What is edk2-nvidia? Is it a version of nvidia's UEFI for Jetson?

How do I change the Jetson's UEFI variables and settings (I'm using a Jetson AGX Orin)? How do I use the information on this [VariableList](https://github.com/NVIDIA/edk2-nvidia/wiki/VariableList#variables) page?

I think my questions may help others too.

Thank you

**Comment by jbrasen** [comment]
edk2-nvidia is the nvidia specific code for supporting UEFI. It is our equivalent of https://github.com/tianocore/edk2-platforms

For UEFI variables most of these are configurable via the UI menu. If you go under the Device Manager -> NVIDIA Configuration.

Some require setting manually or via a device tree overlay

**Comment by PHCLenzi** [comment]
@jbrasen,

Is this "UI menu" accessed by the "Device Manager" inside the standard jetpack? 

Context: I'm trying to understand how flash scripts (from nvidia or Yocto Project for example) change the memory address/device that contains the Kernel, Fileszstem,... .

I noticed that there is "L4TDefaultBootMode" in these[ variables list](https://github.com/NVIDIA/edk2-nvidia/wiki/VariableList#variables) and I would like to know how to access it and change it.

I knew the alternative of adding a device tree overlay with the chosen memory device, but I would like to understand this variable I mentioned above.

Any ideas?

**Comment by jbrasen** [comment]
Yes if press ESC to enter the UEFI menu then the settings are under Device Manager
Device Manager-> NVIDIA Configuration -> L4T Configuration

This changes the way the l4t launcher attempts to boot the OS (grub, from extlinux, etc)
